resources/wizard/style.qss

```diff
@@ -0,0 +1,82 @@
+/*
+    it turned out to be easier to apply styles as "locally" as possible, i.e., just apply where necessary
+
+    we use a dedicated widget as a container for the pages within the wizard window
+    styles will be inherited by all child widgets, so we'd have to guess the original values for all elements which we
+    do not want to style
+
+    the navigation/pagination part is located outside the content widget, so it won't be affected by this stylesheet
+    especially since we prefix all rules with #contentWidget
+
+    note that wizard.* are variables replaced by our own code when loading the stylesheet
```

Comment:
I'm a fan templating stuff with `@foo@`
That we we can also assert at the end that there are no remaining placeholders in the string